### ISSUE TITLE
[Bug]: WYSIWYG Toolbar not attached aynmore/floats after scrolling data object panels

### ISSUE THREAD

**Issue by MB-web** [pimcore/pimcore#16316]
### Pimcore version

11.0

### Steps to reproduce

- Focus a WYSIWYG field within a data object panel that can be scrolled
- Scroll the panel

### Actual Behavior

WYSIWYG Toolbar stays at the same position at the screen and is not correctly attached anymore.

<img width="637" alt="image" src="https://github.com/pimcore/pimcore/assets/35453209/7b35df36-9511-475c-b614-fc65aa8a135d">
<img width="1345" alt="image" src="https://github.com/pimcore/pimcore/assets/35453209/b4c18c22-b36e-4749-9348-ca78be65c6a2">

(Reproducable in https://demo.pimcore.fun/admin/)

### Expected Behavior

WYSIWYG Toolbar stays attached at the top of the editable.

**Comment by robertSt7** [comment]
see https://github.com/pimcore/pimcore/pull/15024

**Comment by MB-web** [comment]
> see #15024

Thank you. It seems there was a similar issue in the previous Pimcore Version where still CKEditor was used and that this one was fixed.

I think the issue I have reported is a new one.

**Comment by robertSt7** [comment]
@MB-web Yes, yours is also valid, but I think we can use a similar approach like in 10.6

**Comment by georgringer** [comment]
if others search for a solution with tinymce, use an editor config

```json
{
    "toolbar1": "undo redo | styles | bold italic | link | numlist bullist | alignleft aligncenter alignright | removeformat | help",

inline: false,

style_formats_merge:false,
style_formats: [
{ title: 'Heading 1', block: 'h1' },
{ title: 'Heading 2', block: 'h2' },
{ title: 'Heading 3', block: 'h3' },
{ title: 'Heading 4', block: 'h4' },
{ title: 'Heading 5', block: 'h5' },
{ title: 'Heading 6', block: 'h6' },
{title : 'big', block : 'div', classes : 'big', exact: true},]
}
```

important is the `inline:false`

**Comment by dpfaffenbauer** [comment]
@georgringer you're doing Pimcore now? :O

**Comment by georgringer** [comment]
@dpfaffenbauer a bit ;) but there are imho more successful implementations than the editor...

**Comment by MB-web** [comment]
@georgringer Thank you for the answer. I remember to also have tried this, but as the setting suggests, the editor is then no longer in inline mode.
If the customer does not mind, its okay, but it was unfortunately no solution for us.
